src/maincurrencydialog.cpp

```diff
@@ -608,15 +612,12 @@ void mmMainCurrencyDialog::OnHistoryUpdate(wxCommandEvent& /*event*/)
     wxString base_currency_symbol;
     if (!Model_Currency::GetBaseCurrencySymbol(base_currency_symbol))
```

Comment:
we should replace this with wxASSERT as symbol is now rerequired (UNIQUE NOT NULL)

Reply:
I thought about this:
```c++
wxASSERT_MSG(Model_Currency::GetBaseCurrencySymbol(base_currency_symbol), "Could not find base currency symbol");
```
replacing all `if() { }`